src/framework/analysisState.ml

```diff
@@ -16,6 +16,9 @@ let svcomp_may_invalid_deref = ref false
 (** Whether an invalid memtrack happened *)
 let svcomp_may_invalid_memtrack = ref false
 
+(** Whether an invalid memcleanup happened *)
+let svcomp_may_invalid_memcleanup = ref false
```

Comment:
The comments are a bit weird: `an invalid memtrack happened` is probably something like `a memleak occurred` and similar for `invalid_memcleanup`.

Reply:
Fixed in https://github.com/goblint/analyzer/pull/1201/commits/7f0b43ccf1db2d6c3bec95ce1ea759dff84e4fba